### PR TITLE
Support for loading sounds from arbitrary directories on Android

### LINKED ISSUE #5: Play sound from URL
Can you add support for playing sound from a URL, eg: http://www.mydomain.com/some/sound1.mp3.

### LINKED ISSUE #8: how to play mp3 file downloaded using react-native-fs
hello, i download a mp3 file using react-native-fs. the file directory on my android device is '/data/data/com.testapp17/files/sample.mp3', but it is unable to play the file using the directory.

### PR DESCRIPTION
**Pull request by jeduan**
Hi! This PR adds support for loading files on Android from arbitrary directories. This fixes #8 and fixes #5.

I tested loading a file from `RNFS.DocumentsDirectory` which worked perfectly.

Thanks!

### CODE REVIEW

**Comment by jeduan** [comment]
Oops just noticed #15

**Comment by DVassilev** [comment]
@zmxv Could this be merged?

**Comment by jeduan** [comment]
@DVassilev in the meantime, you can use my fork by setting on your package.json the dependency like so:
`"react-native-sound": "jeduan/react-native-sound#42bb7dd4da8"`

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ public class MainActivity extends Activity implements DefaultHardwareBackBtnHand
   }
 ```
 
-Save your sound clip files under the directory `android/app/src/main/res/raw`.
+Save your sound clip files under the directory `android/app/src/main/res/raw` or any of the directories under Sound.
```
Comment (zmxv) [inline README.md]: Save your sound clip files under the default directory `android/app/src/main/res/raw`, or any other directory in which case you need to pass the full path into the `Sound` constructor.

File: sound.js [hunk 2]
```diff
@@ -6,11 +6,7 @@ var IsAndroid = (typeof RNSound.setLooping) !== 'undefined';
 var nextKey = 0;
 
 function Sound(filename, basePath, onError) {
-  if (IsAndroid) {
-    this._filename = filename.toLowerCase().replace(/\.[^.]+$/, '');
-  } else {
-    this._filename = basePath ? basePath + '/' + filename : filename;
-  }
+  this._filename = basePath ? basePath + '/' + filename : filename;
```
Comment (zmxv) [inline sound.js]: Please keep the original `IsAndroid` check because file extension is needed on iOS but not on Android.

**Comment by jeduan** [comment]
@zmxv Thanks for the feedback! Just made the changes you suggested

**Comment by zmxv** [comment]
Merged into master. Thank you for the quick response!

**Pull request merged**